### PR TITLE
Header Elevation

### PR DESCRIPTION
**Pull request by arpitBhalla**
**What kind of change does this PR introduce?**
new prop to elevate header component 

**Did you add tests for your changes?**
Yes
**If relevant, did you update the documentation?**
Yes
**Summary**
It looks nice😅

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Comment by flyingcircle** [comment]
I'll be honest. I'm not too much of a fan of this change. There are a few places where having props for different styles might be useful, but usually an app only has a single header. So I just don't see a need to shorthand some styles in a prop that people might just end up wanting to tweak anyways. What are your thoughts?

**Comment by arpitBhalla** [comment]
@flyingcircle it is in Material UI so I thought, it may help user

File: src/header/Header.tsx [hunk 1]
```diff
@@ -144,6 +146,16 @@ const Header: React.FunctionComponent<HeaderProps> = (props: HeaderProps) => {
             alignItems: 'center',
           },
           backgroundColor && { backgroundColor },
+          elevated && {
```
Comment (flyingcircle) [inline src/header/Header.tsx:149]: The style needs to be moved to the bottom in the `StyleSheet`

**Review by flyingcircle — CHANGES_REQUESTED** [review]

File: src/header/Header.tsx (hunk 1 above)
Comment (arpitBhalla) [reply src/header/Header.tsx:149]: Sure Sir

**Comment by flyingcircle** [comment]
I brought this up in the demo app and I thought it just looked a bit strange. What do you think about this instead:
```js
elevatedHeader: {
        shadowColor: '#000',
        shadowOffset: {
            width: 0,
            height: 6,
        },
        shadowOpacity: 0.6,
        shadowRadius: 8.0,
        elevation: 24,
    },
```

**Pull request merged**